app/ts/library/provider.ts

```diff
@@ -69,6 +69,11 @@ export const connectBrowserProvider = async (
 
 	const provider = new BrowserProvider(window.ethereum, 'any')
 
+	const blockCallback = async (blockNumber: number) => {
+		const block = await provider.getBlock(blockNumber)
+		if (block) updateLatestBlock(block, store, blockInfo, signers)
```

Comment:
```suggestion
		if (block) await updateLatestBlock(block, store, blockInfo, signers)
```

Reply:
No need to await, as there is nothing else to execute after in the callback